src/Makefile.am

```diff
@@ -0,0 +1,4 @@
+lib_LTLIBRARIES=libmo_unpack.la
+libmo_unpack_la_SOURCES=convert_float_ibm_to_ieee32.c convert_float_ieee32_to_ibm.c extract_bitmaps.c extract_nbit_words.c extract_wgdos_row.c logerrors.c logerrors.h pack_ppfield.c read_wgdos_bitmaps.ibm.c rlencode.c rlencode.h uascii.c unpack_ppfield.c wgdos_decode_field_parameters.c wgdos_decode_row_parameters.c wgdos_expand_row_to_data.c wgdos_pack.c wgdosstuff.h wgdos_unpack.c
+libmo_unpack_la_LDFLAGS=-version-info 2:5:0
```

Comment:
Sounds like a nice idea. If you know how to do that (I'm still new to autotools so would have to look it up) then a PR would be great.